pkg/operator/certrotation/signer.go

```diff
@@ -223,6 +223,7 @@ func setSigningCertKeyPairSecret(signingCertKeyPairSecret *corev1.Secret, validi
 	signingCertKeyPairSecret.Data["tls.key"] = keyBytes.Bytes()
 	annotations.NotBefore = ca.Certs[0].NotBefore.Format(time.RFC3339)
 	annotations.NotAfter = ca.Certs[0].NotAfter.Format(time.RFC3339)
+	annotations.RefreshPeriod = durationRound(refresh)
```

Comment:
won't this value be overwritten by `EnsureTLSMetadataUpdate` which is called 2 lines below?